node/package.json

```diff
@@ -31,7 +31,7 @@
     "ramda": "^0.26.1",
     "slugify": "^1.2.6",
     "typescript": "3.9.7",
-    "vtex.search-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.search-graphql@0.37.0/public"
+    "vtex.search-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.search-graphql@0.38.0-beta.1/public"
```

Comment:
It would be better if the version was a stable one. You can do this change after the release of the search-graphql

Reply:
Alright!

Reply:
Done